Notifications/notifications_actions_customization.md

```diff
@@ -12,99 +12,111 @@ This document is a starting point for engaging the community and standards bodie
+
+### Use Examples
+
+An incoming call notification from a VoIP PWA without any specified action buttons could be instantiated through a service worker by using:
+
+```javascript
+const title = "Andrew Bares";
+const options = {
+  scenario: "calling",
+  body: "Incoming Call - Mobile",
+  sound: "ringtone"
 }
+
+serviceWorkerRegistration.showNotification(title, options);
 ```
 
-The `"default"` value would play the standard notification audio provided by the platform just once; whereas the `"ringtone"` one would execute a platform-provided audio, suitable for executing in a loop, and keep playing it for the duration of the notification. If the platform is not able to provide a suitable `"ringtone"` audio the User Agent (UA) must fallback to the `"default"` case.
+In this case, the notification would be displayed with a ringtone and also have the default dismiss button as depicted in the figure below.
+
+![Calling notification without provided action buttons](images/no_action.png)
+
+*Figure 2: Calling notification scenario without provided action buttons.*
 
-An incoming call notification from a VoIP Web app could be instantiated through a service worker by using:
+If the web app specifies any action buttons they should show up alongside with the default dismiss buttons and, if the platform allows, they should have colors different from the dismiss button. A PWA would be able to send a `"incoming-call"` notification with colored buttons by means of a service worker using:
 
 ```javascript
 const title = "Incoming call";
 const options = {
-  body: "John D.",
+  scenario: "calling",
+  title: "Andrew Bares",
+  body: "Incoming Call - Mobile",
   sound: "ringtone",
   actions: [
     {
         action: "accept-audio-call",
-        title: "Accept",
-        icon: '/app/images/phone.png',
-        style: "success"
+        title: "audio",
+        icon: "https://web.app.com/assets/accept_audio_call.png"
     },
     {
         action: "accept-video-call",
-        title: "Video",
-        icon: '/app/images/camera.png',
-        style: "success"
-    },
-    {
-        action: "decline-call",
-        title: "Decline",
-        icon: '/app/images/decline.png',
-        style: "critical"
-    }    
+        title: "video"
+        icon: "https://web.app.com/assets/accept_video_call.png"
+    }
   ]
 }
 
 serviceWorkerRegistration.showNotification(title, options);
 ```
 
+The notification should look like this:
+
+![Calling notification with action buttons](images/notification_with_actions.png)
+
+*Figure 3: Calling notification scenario with action buttons.*
+
+### Extension Scope
+
+The Notifications API is a [powerful feature](https://w3c.github.io/permissions/#dfn-powerful-feature) and, given that the extensions proposed in this explainer could be potentially abused, they should only be made available for PWAs. Therefore, the `Notification.scenario` and `Notification.sound` properties should be ignored unless a PWA is setting them.
```

Comment:
Remove reference to `Notification.sound`  here.

Reply:
Removed. Thanks for pointing out!